src/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portafolio Main Page</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap" rel="stylesheet">
+</head>
+<body>
+<header class="navbar">
+    <a class="logo" href="">My logo</a>
+    <nav>
+        <img class="enabled" src="../assets/Enabled.png" alt="">
+    </nav>
```

Comment:
- [ ] As suggested by the previous reviewer, kindly increase the size of your hamburger to match the figma design 👍🏽 

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/219900175-a9796a0a-8385-473f-b432-e5b08de09ba6.png)

figma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/219900195-3e375da8-8832-40e2-94ac-17e052e10aca.png)